1.0/docs/start/psk/set-up.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Set up
+subtitle: Using Polymer Starter Kit
+---
+
+{% include toc.html %}
+
+The Polymer Starter Kit is a complete starting point for a Polymer 1.0 
+application. It includes:
+
+*   A [Material Design][md] layout
+*   Routing with Page.js
+*   Web component unit testing
```

Comment:
Can we go with "Unit testing with [web-component-tester](link to repo)." And periods.